src/node_options.cc

```diff
@@ -546,6 +546,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::prof_process);
   // Options after --prof-process are passed through to the prof processor.
   AddAlias("--prof-process", { "--prof-process", "--" });
+  AddOption("--experimental-import-map",
+            "set the path to an import map.json",
```

Comment:
```suggestion
            "import map to use for the module resolution algorithm",
```

Reply:
To avoid suggesting that it might be used _with_ the module resolution algorithm, as opposed to in place of it:

```suggestion
            "import map to use for module resolution",
```

Reply:
How about `"load an import map"`. Because I don't think either of these are true:

1. import map to use for the module resolution algorithm: it does replace the algo
2. import map to use for module resolution: it does not require that it resolve or fully replace the algo

What it actually does is alter the URL and then pass it back to the module resolution algo. If it fully resolves to a file, then it might not do more, but it doesn't "replace" it.  An example, if you did `"imports": { "foo/": "bar/" }` it will still go back to `defaultResolve` inside `bar/` to process `exports` and `main` and then use local file resolution from there.

Reply:
Calling this resolved, I pushed a version that is a combination of these suggestions I think.